File: ProgramFour/code.py
def checkTrinagle(a,b,c):
    if a == b and b == c and c == b:
        return "Equilateral Triangle"
    elif a == b or b == c or c == a:
        return "Isosceles Triangle"
    else: 
        return "Scalene Triangle"

print(checkTrinagle(1,2,3))
print(checkTrinagle(3,2,3))
print(checkTrinagle(3,3,3))

File: ProgramFour/code.js
const checkTraingle = (a,b,c) => {
    if(a === b && b === c && c === a) return 'Equalateral Triangle'
    if(a === b || b === c || c === a) return 'Isosceles Triangle'
    return "Scalene Triangle"
}

console.log(checkTraingle(1,2,3));
console.log(checkTraingle(3,2,2));
console.log(checkTraingle(3,3,3));